HACKING.md

```diff
@@ -19,6 +19,22 @@ Note: To build the base and release images, run:
 
     $ hack/build-base-images.sh
 
+Once a release has been created, it can be pushed:
+
+    $ hack/push-release.sh
+
+To cut an official tag release, we generally use the images built by [ci.openshift.redhat.com](https://ci.openshift.redhat.com)
+under the openshift3_ami job.
+
+1. Create a new git tag `git tag v0.X.X -a -m "v0.X.X" HEAD`
+2. Push the tag to GitHub `git push origin --tags`
```

Comment:
GitHub OpenShift

----- Original Message -----
> > @@ -19,6 +19,22 @@ Note: To build the base and release images, run:
> >  
> >      $ hack/build-base-images.sh
> >  
> > +Once a release has been created, it can be pushed:
> > +
> > +    $ hack/push-release.sh
> > +
> > +To cut an official tag release, we generally use the images built by
> > [ci.openshift.redhat.com](https://ci.openshift.redhat.com)
> > +under the openshift3_ami job.
> > +
> > +1. Create a new git tag `git tag v0.X.X -a -m "v0.X.X" HEAD`
> > +2. Push the tag to GitHub `git push origin --tags`
> 
> Is origin openshift or smarterclayton?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/864/files#r24084806